src/ACEtk/block/details/BaseDistributionData/src/ctor.hpp

```diff
@@ -0,0 +1,48 @@
+BaseDistributionData() = default;
```

Comment:
[clang-tidy - recommended]
All constructors are defined, but there is no destructor.